config/application.rb

```diff
@@ -33,5 +33,15 @@ class Application < Rails::Application
       g.test_framework :rspec
       g.fixture_replacement :factory_bot, dir: 'spec/factories'
     end
+
+    # Only loads a smaller set of middleware suitable for API only apps.
+    # Middleware like session, flash, cookies can be added back manually.
+    # Skip views, helpers and assets when generating a new resource.
+    config.api_only = true
```

Comment:
what happens with ActiveAdmin?

Reply:
It just works, but after we add these https://github.com/rootstrap/rails_api_base/pull/253/files#diff-b1fe55db50c712fef0673345e5b9c0d9R42

Reply:
and is the order of those middlewares fine? it doesn't even matter?

Reply:
I guess the order matters but maybe not in this case. I tried it locally and everything worked fine.

Reply:
Maybe they matter, and maybe I added the flash deliberately at the end because it uses the cookie (?) haha _sitedigolaverdadtemiento.com_

Reply:
maybe we should insert them in the rails default order?

Reply:
Ok sounds a good idea, I'll investigate about that ordering

Reply:
This is the default order https://guides.rubyonrails.org/rails_on_rack.html#inspecting-middleware-stack